pkg/helm/actions/install_chart.go

```diff
@@ -45,6 +140,35 @@ func InstallChart(ns, name, url string, vals map[string]interface{}, conf *actio
 	if ch.Metadata.Name != "" && ch.Metadata.Version != "" {
 		metrics.HandleconsoleHelmInstallsTotal(ch.Metadata.Name, ch.Metadata.Version)
 	}
-
+	// remove all the tls related files created by this process
+	defer func() {
+		if os.Getenv("HELM_CLEANUP") == "0" {
```

Comment:
Let's move this `os.Getenv` call to a function argument.

Reply:
Instead of using os,.Getenv i am using a bool param in the function call.